recipe/meta.yaml

```diff
@@ -28,10 +28,19 @@ requirements:
     - nsis >=3.01  # [win]
 
 test:
-  imports:
-    - constructor
+  source_files:
+    - tests
+    - scripts
+    - examples
+  requires:
+    - pillow >=3.1
+    - pytest
   commands:
+    - pytest -v tests
+    - python scripts/run_examples.py
```

Comment:
Just adding those because we previously had the `constructor --test .` here.
(If they cause trouble, we can remove `- examples`/`- pillow >=3.1`/`- python scripts/run_examples.py` again.)

Reply:
seems run_examples.py fails on aarch64